tomopy/misc/morph.py

```diff
@@ -73,7 +73,7 @@
 LIB_TOMOPY = extern.c_shared_lib('libtomopy')
 
 
-def pad(arr, axis, npad=None, mode='constant', **kwargs):
+def pad(arr, axis, npad=None, mode=str('constant'), **kwargs):
```

Comment:
It would read better to put the `str` in the call to pad rather than in the signature.